### PR TITLE
Documentation v1

### PR DESCRIPTION
**Pull request by estemendoza**
Starting with the documentation process

### CODE REVIEW

File: documentation/main.md [hunk 1]
```diff
@@ -0,0 +1,280 @@
...
+Here is an example on how to do this:
+
+```python
+# test_step.py
+from marvin import Step
+
+from utils.db_connection import CheckDBConnection
+
+
+class TestStep(Step):
```
Comment (estemendoza) [inline documentation/main.md]: Didn't really get it what you mean with this

Comment (argos83) [inline documentation/main.md]: I meant the class name `TestStep`, could be renamed to `ConnectToDatabase`

Comment (estemendoza) [inline documentation/main.md]: Ok, great

**Comment by estemendoza** [comment]
@argos83 can you check the changes that you suggested? also, don't know really how to fix the error on code climate

**Comment by argos83** [comment]
> @argos83 can you check the changes that you suggested? also, don't know really how to fix the error on code climate

It says: `Error: Missing parentheses in call to 'print' (<unknown>, line 14)</unknown> NEW`
```
print "#### CUSTOMER LOGGER - TEST STARTING ####"
```

Change it to `print("### CUSTOM LOGGER....")` also notice there's a typo (should be CUSTOM instead of CUSTOMER, right?)

File: docs/main.md [hunk 2]
```diff
@@ -0,0 +1,291 @@
...
+from marvin import TestScript
+
+
+class CreateUser(TestScript):
+    """Step for creating a user specifying a user type"""
+
+    NAME = "Create User"
+
+    def setup(self, data):
+        print "Using username {}".format(data["username"])
```
Comment (argos83) [inline docs/main.md]: RIght, also in the docs let's be python3 compliant. let's use the print function instead of the print statement. I.e `print(...)`

**Pull request merged**